specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json

```diff
@@ -3191,6 +3225,20 @@
       },
       "description": "Application list operation result."
     },
+    "ServicePrincipalObjectResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "Service Principal Application Object Id"
+        },
```

Comment:
isnt the value a array? from the docs you shared it looks like an array of strings